package org.hartz.hartz_backend.exception.userExceptions;

public class EmailTakenException extends RuntimeException {
}
